shared/router-v2/router.desktop.tsx

```diff
@@ -8,21 +8,36 @@ import {createLeftTabNavigator} from './left-tab-navigator.desktop'
 import createNoDupeStackNavigator from './stack'
 import {NavigationContainer} from '@react-navigation/native'
 import {modalRoutes, routes, loggedOutRoutes, tabRoots} from './routes'
+import type {RouteMap} from '../util/container'
 import * as Shim from './shim.desktop'
 import * as Common from './common.desktop'
 import {HeaderLeftCancel} from '../common-adapters/header-hoc'
 
 export const headerDefaultStyle = Common.headerDefaultStyle
 const Tab = createLeftTabNavigator()
 
+type DesktopTabs = typeof Tabs.desktopTabs[number]
+
+const tabRootsVals = Object.values(tabRoots)
+// we don't want the other roots in other stacks
```

Comment:
while debugging i was getting into a state where the chat inbox was on top of profile which is a thing we basically never want